### PR TITLE
fix: integration tests execution

### PR DESCRIPTION
**Pull request by loonatic**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 41s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 34s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 4m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 21m 43s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1367/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 21m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1367/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: mondrian/src/it/java/mondrian/test/Olap4jTckTest.java [hunk 1]
```diff
@@ -57,9 +57,15 @@ public static TestSuite suite() {
         final TestSuite suite = new TestSuite();
 
         suite.setName("olap4j TCK");
-        suite.addTest(createMondrianSuite(connStr, false));
+
+        //These suits are comments due to an update from commons-dbcp to commons-dbcp2 in this commit bellow.
```
Comment (davidmsantos90) [inline mondrian/src/it/java/mondrian/test/Olap4jTckTest.java:61]: `comments` should be `commented`.

also, please add a whitespace at the start of every comment (`// foo`)

**Review by davidmsantos90 — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 20m 55s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1367/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 20m 27s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1367/8/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by afrjorge — APPROVED** [review]

**Pull request merged**